Formula/javacc.rb

```diff
@@ -1,27 +1,27 @@
 class Javacc < Formula
-  desc "Java parser generator"
+  desc "The most popular parser generator for use with Java applications"
```

Comment:
Let's avoid “most popular”. In fact, let's keep the previous description.

Reply:
- We would like the strapline to be consistent with the new project website we are launching at https://javacc.github.io/javacc/
- The docs were out of date and have been completely rewritten in markdown for 7.0.5.
- Docs have moved from “doc” directory to “docs” per convention for hosting on GitHub Pages
- Docs, examples, and lib have been moved under libexec to be consistent with projects like Flink.
- Previous version 7.0.4 was created without consultation with the JavaCC project team and will not work with any version after 7.0.4.

Reply:
- We would like the strapline to be consistent with the new project website we are launching at https://javacc.github.io/javacc/
- The docs were out of date and have been completely rewritten in markdown for 7.0.5.
- Docs have moved from “doc” directory to “docs” per convention for hosting on GitHub Pages
- Docs, examples, and lib have been moved under libexec to be consistent with projects like Flink.
- Previous version 7.0.4 was created without consultation with the JavaCC project team and will not work with any version after 7.0.4.